google/cloud/forseti/services/base/config.py

```diff
@@ -148,15 +149,50 @@ def get_storage_class(self):
 
 
 class InventoryConfig(AbstractInventoryConfig):
+    # pylint: disable=too-many-instance-attributes
     """Implements composed dependency injection for the inventory."""
 
+    @staticmethod
+    def _filter_valid_resources(excluded_resources):
+        """Filter valid excluded resources.
+
+        Args:
+            excluded_resources (list): A list of excluded resources to
+                validate against.
+
+        Returns:
+            list: List of valid resources to exclude.
+        """
+        valid_resource_patterns = [
+            'organization/\\d+',
+            'folder/\\d+',
+            'project/.+'
```

Comment:
Use the project id regex. "It must be 6 to 30 lowercase letters, digits, or hyphens. It must start with a letter. Trailing hyphens are prohibited."

[a-z][-a-z0-9]{4,28}[a-z0-9]

Reply:
updated